l10n.js

```diff
@@ -197,7 +202,6 @@ document.webL10n = (function(window, document, undefined) {
       var dictionary = {};
 
       // token expressions
-      var reBlank = /^\s*|\s*$/;
```

Comment:
Don't remove the trimming part without justification. This alone increases the likelihood of breaking compatibility with existing localization files, if anyone ever upgrades their copy of webL10n.